.github/workflows/deploy-staging.yaml

```diff
@@ -0,0 +1,25 @@
+name: Build and Deploy
+
+on:
+  push:
+    branches: [develop]
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v3
+
+      - name: Install and Build 🔧
+        run: |
+          npm install
+          npm run export
+
+      - run: touch ./out/.nojekyll
+
+      - name: Deploy 🚀
+        uses: JamesIves/github-pages-deploy-action@4.1.5
+        with:
+          branch: gh-pages # The branch the action should deploy to.
+          folder: out # The folder the action should deploy.
```

Comment:
we should set this  next.config `distDir` property. I think the default build dir is `.next`

Reply:
If you build it goes in `.next/` (with SSR and server and so on)
If you export, it goes in `out/` (only static files for static deployment)

Reply:
nice thanks!